docs/modules/setup/pages/install.adoc

```diff
@@ -64,7 +67,14 @@ docker run -p8080:8000 -d yuzutech/kroki
 == Using docker-compose
 
 You can use `docker-compose` to run multiple containers.
-Here's an example where we start all the containers:
+
+Assumptions:
+
+* [x] {uri-podman-get-started}[Podman] is installed on your machine.
+* [x] {uri-podman-compose-install}[Podman-compose] is installed on your machine.
```

Comment:
I believe that this section should be under "Using Podman" right?
The assumption in the "docker-compose" is:

```suggestion
Assumptions:

* [x] {uri-docker-compose-install}[Docker Compose] is installed on your machine 
```